examples/gsph/run_disc_1sink.py

```
@@ -0,0 +1,265 @@
+====================================================================
+
+"""
+
+import os
+
+import numpy as np
+
+import shamrock
+
+outputdir = "_to_trash/gsph_onesink/"
+os.system("mkdir -p " + outputdir)
```

Ruff (0.16.2)

[error] Starting a process with a shell, possible injection detected

(S605)